spec/mock_data_spec.rb

```diff
@@ -0,0 +1,86 @@
+require 'rspec'
+require './data/mock_data'
+
+describe MockData do
+  describe '#get_mock_merchants' do
+    it 'returns mock data as an array of hashes' do
+      mocks = MockData.get_mock_merchants
+
+      expect(mocks).to be_instance_of Array
+      expect(mocks.length).to eq 10
+      expect(mocks.first).to be_instance_of Hash
+    end
+
+    it 'returns mock data of merchants with data' do
+      mocks = MockData.get_mock_merchants(number_of_mocks: 2)
+      mocked_merchant = mocks.first
+      expect(mocks.length).to eq 2
+      expect(mocked_merchant[:name]).to eq 'Merchant 0'
+      expect(mocked_merchant[:id]).to eq 0
+      expect(mocked_merchant[:created_at]).to match /\d{4}-\d{2}-\d{2}/
+      expect(mocked_merchant[:updated_at]).to match /\d{4}-\d{2}-\d{2}/
+    end
+  end
+
+  describe '#get_mock_items' do
+    it 'returns mock data as an array of hashes' do
+      mocks = MockData.get_mock_items
+
+      expect(mocks).to be_instance_of Array
+      expect(mocks.length).to eq 10
+      expect(mocks.first).to be_instance_of Hash
+    end
+
+    it 'returns mock data of items with data' do
```

Comment:
should we be testing the different parameters here? Doesn't seem like we've tested that it can return data with different numbers of merchants or a specific price, etc. ?